src/components/SidePanes/UsersPane.test.jsx

```diff
@@ -112,9 +112,6 @@ describe('UsersPane', () => {
     const startAddButton = wrapper.find('.usersPane-startAdd');
     startAddButton.simulate('click');
 
-    const select = wrapper.find('select');
```

Comment:
Is this not possible to keep anymore? I could see this check still having value.

Reply:
I was actually able to select the option rather than the entire select field.

If you're curious, follow up to your suggestion of debugging the wrapper: I was running into issues using `wrapper.find()` because the InputSelect component from Paragon is actually labeled as `<asInput(withDeprecatedProps(InputSelect)) />`, so neither `wrapper.find('select')` or `wrapper.find('InputSelect')` was doing the trick. Selecting the option directly is more straight forward, and will (hopefully) help if/when we switch to Paragon V2.